cloud-provider-azure-1.31.yaml

```diff
@@ -0,0 +1,75 @@
+package:
+  name: cloud-provider-azure-1.31
+  version: 1.31.1
+  epoch: 0
+  description: Cloud provider for Azure
+  copyright:
+    - license: Apache-2.0
+
+var-transforms:
+  - from: ${{package.version}}
+    match: ^(\d+\.\d+)\.\d+$
+    replace: "$1"
+    to: major-minor-version
+
+data:
+  - name: cloud-provider-azure-packages
+    items:
+      cloud-controller-manager: Controller manager for Azure Cloud
+      cloud-node-manager: Node manager for Azure Cloud
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/kubernetes-sigs/cloud-provider-azure
+      tag: v${{package.version}}
+      expected-commit: 301bb5195472aa76bb36e24726c197ccbb19cfca
+
+subpackages:
+  - range: cloud-provider-azure-packages
+    name: cloud-provider-azure-${{range.key}}-${{vars.major-minor-version}}
+    description: ${{range.value}}
+    pipeline:
+      - uses: go/build
+        with:
+          packages: ./cmd/${{range.key}}
+          deps: github.com/golang-jwt/jwt/v4@v4.5.1
```

Comment:
this deps will be bumped by go/bump pipeline which is a separate pipeline. You'll have to do something like this: 

```bash
  - uses: go/bump
    with:
      deps: github.com/golang-jwt/jwt/v4@v4.5.1
```

Reply:
I see, thanks Anurag!